static/less/humhub.less

```diff
@@ -250,3 +250,10 @@ ul.tag_input {
   line-height: 8px;
 }
 
+
+span.likeLinkContainer .like.disabled, span.likeLinkContainer .unlike.disabled {
```

Comment:
humhub.less not the best place for this, rather use base.less, which is also not ideal but I did not find a better place either.

Reply:
As you please, I have moved it.
If you would find a better place to store it, just let me know, and I will move it to another place if it is needed.